samples/unpeelingOnion/src/onion/ecommerce/application/shoppingCarts/index.ts

```diff
@@ -0,0 +1,62 @@
+import { MongoClient } from 'mongodb';
+import { registerCommandHandler } from '#core/commands';
+import { registerQueryHandler } from '#core/queries';
+import { OpenShoppingCart } from 'src/onion/ecommerce/application/shoppingCarts/commands/shoppingCarts/openShoppingCart';
+import { GetShoppingCartById } from 'src/onion/ecommerce/application/shoppingCarts/queries/getShoppingCartById';
+import { AddProductItemToShoppingCart } from 'src/onion/ecommerce/application/shoppingCarts/commands/shoppingCarts/addProductItemToShoppingCart';
+import { RemoveProductItemFromShoppingCart } from './commands/shoppingCarts/removeProductItemFromShoppingCart';
+import { ShoppingCartMapper } from './mappers/shoppingCartMapper';
+import { OpenShoppingCartHandler } from './commandHandlers/openShoppingCartHandler';
+import { AddProductItemToShoppingCartHandler } from './commandHandlers/addProductToShoppingCartHandler';
+import { RemoveProductItemFromShoppingCartHandler } from './commandHandlers/removeProductFromShoppingCartHandler';
+import { GetShoppingCartByIdHandler } from './queryHandlers/getShoppingCartByIdQueryHandler';
+import { MongoDBShoppingCartRepository } from 'src/onion/ecommerce/infrastructure/shoppingCarts/shoppingCartRepository';
+import { ConfirmShoppingCart } from './commands/shoppingCarts/confirmShoppingCart';
+import { ConfirmShoppingCartHandler } from './commandHandlers/confirmShoppingCart';
+import { GetCustomerShoppingHistory } from './queries/getCustomerShoppingHistory';
+import { GetCustomerShoppingHistoryHandler } from './queryHandlers/getCustomerShoppingHistoryHandler';
+import { CustomerShoppingHistoryMapper } from './mappers/customerShoppingHistoryMapper';
+import { EventBusFactory } from '#core/events';
+
+const registerHandlers = (mongo: MongoClient) => {
```

Comment:
This will be messy in the future. The better option is to prepare the configuration and facade for each module.

Reply:
Yeah, this class is configuration for the module. Do you have an example of a facade you’re thinking of?